cliquet/__init__.py

```diff
@@ -26,7 +26,7 @@
 DEFAULT_SETTINGS = {
     'cliquet.backoff': None,
     'cliquet.batch_max_requests': 25,
-    'cliquet.cache_backend': 'cliquet.cache.redis',
+    'cliquet.cache_backend': 'cliquet.cache.memory',
```

Comment:
I'm sorry but I don't follow you here.

If this PR is supposed to fix the handling of default settings, why do we have to put the same default backend settings in Cliquet and Kinto ?